src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
sceneEl might be needed in other part of the function and you won't see is defined. If it's only needed within the function make a local variable or create a closure to hide ir. There's a usual case to define a variable in a function that it's only use in a small section of the body. I still think it has to be moved up.